### DISCUSSION TITLE
Does Rodauth Rails support SSO?

### DISCUSSION THREAD

**Discussion by mdodell** [janko/rodauth-rails#206] [Q&A]
I'm looking into trying to integrate Active Directory for SSO, and I was wondering if Rodauth supports that. I didn't see anything referenced in the documentation, but I could have missed it!

**Comment by janko** [comment]
Active Directory is not supported directly, but according to the docs, you can override Rodauth to use LDAP Authentication – https://github.com/jeremyevans/rodauth#label-Using+LDAP+Authentication.
